client/my-sites/upgrades/checkout-thank-you/header/index.jsx

```diff
@@ -42,13 +43,27 @@ const CheckoutThankYouHeader = React.createClass( {
 		}
 
 		if ( this.props.primaryPurchase ) {
-			return this.translate(
-				"You will receive an email confirmation shortly for your purchase of %(productName)s. What's next?", {
+			if ( this.props.isFreeTrial ) {
+				return this.translate( "We hope you enjoy {{strong}}%(productName)s{{/strong}}. What's next? Take it for a spin!", {
 					args: {
-						productName: this.props.primaryPurchase.productName
+						productName: this.props.productName
+					},
+					components: {
+						strong: <strong/>
+					}
+				} );
+			} else {
+				return this.translate(
```

Comment:
We have a [linting rule](https://github.com/Automattic/wp-calypso/blob/master/.eslintrc.js#L34) for unnecessary else clauses like this.